### PR TITLE
#273: Add support for Docker port bindings

### PR DESCRIPTION
**Pull request by dbryant4**
This addresses #273 by passing `port_bindings` from `molecule.yml` to the docker-py `create_container()` method.

More information about the supported datatype can be found in the [docker-py docs](https://github.com/docker/docker-py/blob/master/docs/port-bindings.md)

### CODE REVIEW

File: docs/source/provisioners.rst [hunk 1]
```diff
@@ -47,15 +50,17 @@ Docker Example
...
-            image: ubuntu
-            image_version: latest
-            privileged: True
-          - name: foo-02
-            ansible_groups:
-              - group2
-            image: ubuntu
-            image_version: latest
-            registry: testhost:5323
+          image: ubuntu
```
Comment (dbryant4) [inline docs/source/provisioners.rst]: The formatting of this sample `molecule.yaml` file was a little confusing.

**Comment by retr0h** [comment]
👍 @dbryant4 Nice addition.  

This won't be released to pypi until #275 is resolved.

**Comment by alop** [comment]
very nice 👍

**Comment by dbryant4** [comment]
Is there a way a travis ci build can be kicked off for this branch? I like seeing the green check mark for PRs :)

**Comment by retr0h** [comment]
Yeah, we haven't implemented travis, due to us having to grant travis access to our org.  Lot of secret sauce :)

We will tie it into our own CI at some point.

**Comment by rgreinho** [comment]
Well, I do have travis set up for [my clone], but it looks like you cannot have the green mark until #275 is solved.

But just going through the code, it looks good.

[my clone]: https://travis-ci.org/rgreinho/molecule/builds/146770235

**Comment by kireledan** [comment]
Nice!

**Pull request merged**